common/djangoapps/custom_reports/api/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
+init.py file for api
+"""
+
+# Flag for enabling Threading in report generation
+ENABLE_THREADING_IN_RG = True

```

Comment:
should it be a microsite setting ?

Reply:
I assume threading is a server level decision? We'd probably use threading unless the server didn't support it? So probably not a microsite setting? 

I really don't know though.

Reply:
I dont know the context for this django app, but this settings should be configurable (django settings, microsite settings, waffle flag.....etc) rather that change the code to modify the behavior.

Reply:
understood. I'll make them change that.